zingolib/src/wallet/transactions.rs

```diff
@@ -403,25 +402,29 @@ impl TransactionMetadataSet {
     pub fn mark_txid_nf_spent(
```

Comment:
I don't understand what this method name means.

Reply:
Yeah, me neither. Renamed locally, haven't pushed yet.